### ISSUE TITLE
GPU underutilized on Mac and AMD Navi

### ISSUE THREAD

**Issue by gunnarmein-ucbischool** [plaidml/plaidml#823]
Hello, thank you for providing this great platform, hopefully you will find a way to keep a Keras API alive this year.

I am evaluating CIFAR10 through Keras on a MacBook Pro with an 8-core i9 CPU and an AMD Radeon 5500m (Navi) GPU. CPU beats GPU by 10%. With the GPU in plaidML, GPU utilization is only 0.5%, the machine is not breaking a sweat. On CPU with TensorFlow, it is correctly using 8 cores. How can I help assess this bottleneck?

Thanks!

**Comment by gunnarmein-ucbischool** [comment]
I also tried a training round on resnet - this time through keras->tf->ngraph-bridge->plaidml, TF on CPU beats this by more than a factor of 2.

**Comment by dmckinno** [comment]
@gunnarmein-ucbischool, I recently started using PlaidML and observed a few things that may help:

1) How are you measuring GPU utilization? The Apple/AMD GPU utilization tools that I have experimented with don't seem to work reliably.

2) My initial GPU training attempts were also slower. I needed to adjust training hyperparameters to take advantage of the parallel computation. I have the same machine as you and see ~5x speed up over CPU. This isn't great but certainly helps with experimentation.
